tests/collections/testsectionlist.ts

```diff
@@ -0,0 +1,110 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2015, Phosphor Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+module tests {
+
+  import SectionList = phosphor.collections.SectionList;
+
+
+
+describe('phosphor.collections - sectionlist', () => {
+
+  describe('insert()', () => {
+
+    it('should return true on success', () => {
+      var obj = new SectionList();
+      expect(obj.size).to.be(0);
+      expect(obj.count).to.be(0);
+      obj.insert(0, 100, 10);
+      expect(obj.size).to.be(1000);
+      expect(obj.count).to.be(100);
+    });
+
+    it('should create heterogenous list', () => {
```

Comment:
'heterogeneous' typically refers to types, maybe 'list with variable sized sections' is more appropriate